strawberryfields/ops.py

```diff
@@ -859,6 +878,40 @@ def _apply(self, reg, backend, **kwargs):
         backend.prepare_ket_state(ket, *reg)
 
 
+class GKP(Preparation):
+    r"""Prepare a mode in a finite energy Gottesman-Kitaev-Preskill (GKP) state.
+
+    In their ideal form, square lattice GKP states are linear combinations of position eigenkets :math:`\ket{\cdot}_q`
+    spaced every :math:`\sqrt{\pi\hbar}`. Finite energy GKPs are attained by applying the Fock damping
+    operator :math:`e^{-\epsilon\hat{n}}` to the ideal states.
+
+    GKP states are qubits, with the qubit state defined by:
+
+    .. math::
+
+    \ket{\psi}_{gkp} = \cos\frac{\theta}{2}\ket{0}_{gkp} + e^(-i\phi)\sin\frac{\theta}{2}\ket{1}_{gkp}
+
+    where the computational basis states are :math:`\ket{\mu}_{gkp} = \sum_{n} \ket{(2n+\mu)\sqrt{\pi\hbar}}_{q}`.
+
+    Square lattice GKPs have Wigner functions with peaks arranged on a square lattice, whereas alternative
+    lattices, such has hexagonal GKPs, can be obtained by applying symplectic transformations to the
+    square lattice GKPs.
+
+    Args:
+        state (list): [theta,phi] for qubit definition above
+        epsilon (float): finite energy parameter of the state
+        cutoff (float): if using the 'real' representation, this determines
+                how many terms to keep
+        desc (string): 'real' or 'complex' reprsentation
+        shape (string): 'square' lattice or otherwise
```

Comment:
Same comment here as above. Should perhaps write something more akin to "the shape of the lattice".